### ISSUE TITLE
cli executable geojson2osm

### ISSUE THREAD

**Issue by andrewharvey** [DenisCarriere/geojson2osm-es6#1]
Like https://github.com/Rub21/geojson2osm it would be great if this would run as a cli executable program out of the box.

**Comment by DenisCarriere** [comment]
Agreed! However this library isn't completed yet, had issues with OSM Elements that has the author as `<don>`. Once the library is fully tested and running smoothly I would be a CLI.
